unified-scheduler-pool/src/lib.rs

```diff
@@ -258,6 +270,7 @@ impl BankingStageHelper {
     }
 
     pub fn generate_task_ids(&self, count: usize) -> usize {
+        assert!(count > 0);
```

Comment:
Can you explain why this is necessary?

Reply:
agree that is true - but it'd be unused afaict

I guess what I'm a bit confused about is that it seems like this would not cause issues if count were 0 - since there would be no transactions to insert in that case anyway?

Reply:
hmm, you have a point; i just removed this assert altogether, i was too carried away with the fixed-number invocation of `generate_task_ids(1)`. Come to think of it, it's called `generate_task_ids(batch.len())` in `core/src/banking_stage/unified_scheduler.rs`: e15dd903d9d3767fd26e91f90dd2b100069435ac